src/shared/lib/CivicUtils.ts

```diff
@@ -196,3 +39,51 @@ export function getCivicCNAVariants(
     }
     return geneVariants;
 }
+
+export async function fetchCivicGenes(
```

Comment:
all of these functions can be non-async that just return a promise. i think that's more typical to how we've been doing things